### PR TITLE
Add dispatch key example

### PR DESCRIPTION
**Pull request by serhii-kozionov**
We are often asked for the key event dispatching examples in JxBrowser Support.
This PR adds `DispatchKeyEventExample`. Later this example will be added to the support portal using the  `embedcode`.

### CODE REVIEW

**Comment by serhii-kozionov** [comment]
@vladimir-ikryanov PTAL

File: input/src/main/java/DispatchKeyEventExample.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+import java.awt.event.KeyEvent;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.util.concurrent.CountDownLatch;
+import javax.swing.JFrame;
+import javax.swing.SwingUtilities;
+import javax.swing.WindowConstants;
+
+public final class DispatchKeyEventExample {
```
Comment (vladimir-ikryanov) [inline input/src/main/java/DispatchKeyEventExample.java:42]: Add Javadoc.

File: input/src/main/java/DispatchKeyEventExample.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            });
+            frame.setDefaultCloseOperation(WindowConstants.DISPOSE_ON_CLOSE);
+            frame.add(view);
+            frame.setSize(800, 500);
+            frame.setLocationRelativeTo(null);
+            frame.setVisible(true);
+        });
+
+        loadHtmlAndWait(browser, HTML);
+        dispatchKeyEvent(browser, (char) KeyEvent.VK_H, KeyCode.KEY_CODE_H);
```
Comment (vladimir-ikryanov) [inline input/src/main/java/DispatchKeyEventExample.java:67]: Extract this to a separate method for better reading.

**Review by vladimir-ikryanov — CHANGES_REQUESTED** [review]

**Comment by serhii-kozionov** [comment]
@vladimir-ikryanov PTAL

**Review by vladimir-ikryanov — APPROVED** [review]
LGTM

**Pull request merged**